components/ads130e08/ads130e08.c

```diff
@@ -0,0 +1,689 @@
+/*
+ * The MIT License (MIT)
```

Comment:
the source code says it's MIT, but the license file says otherwise. which is the one you intended?

Reply:
MIT ... fixed on commit #b4e1988